src/main/java/com/terraformersmc/modmenu/gui/ModMenuOptionsScreen.java

```diff
@@ -21,11 +21,17 @@ public ModMenuOptionsScreen(Screen previous) {
 
 	@Override
 	protected void init() {
-		this.list = this.addDrawableChild(new OptionListWidget(this.client, this.width, this.height, this));
+		this.list = this.addDrawableChild(new OptionListWidget(this.client, this.width, this));
 		this.list.addAll(ModMenuConfig.asOptions());
 		super.init();
 	}
 
+	@Override
+	protected void addOptions()
+	{
```

Comment:
Nitpick: braces should be on the same lines as keywords.

```suggestion
	protected void addOptions() {
```